documentation/repository-configuration/index.md

```diff
@@ -27,14 +27,83 @@ but for production environment you need something real.
 
 First, choose proper JaversRepository implementation.
 Currently, JaVers supports the following databases: **MongoDB**, **H2**, **PostgreSQL**, **MySQL**, **MariaDB**,
-**Oracle** and **Microsoft SQL Server**. 
+**Oracle**, **Microsoft SQL Server** and **Redis**.
 
 In JaVers 5.2.4, we have added experimental support for **Amazon DocumentDB** which claims to be
 almost fully compatible with MongoDB.
 
 **Hint.** If you are using Spring Boot, just add one of our
 [Spring Boot starters for Spring Data](/documentation/spring-boot-integration/)
-and let them automatically configure and boot a JaVers instance with proper JaversRepository implementation. 
+and let them automatically configure and boot a JaVers instance with proper JaversRepository implementation.
+
+<h2 id="redis-db">Redis</h2>
```

Comment:
maybe it's wort to add some hints here about the Redis data persistence model , which model would be recommended for Javers snapshots stored in Redis?
https://redis.io/docs/latest/operate/oss_and_stack/management/persistence/

Reply:
Docs updated to include recommendations for the Redis persistence model.